runtime/src/lib.rs

```diff
@@ -159,6 +192,21 @@ pub const EPOCH_DURATION_IN_BLOCKS: BlockNumber = 1 * HOURS;
 pub const UNIT: Balance = 1_000_000_000_000;
 pub const MILLIUNIT: Balance = 1_000_000_000;
 pub const MICROUNIT: Balance = 1_000_000;
+
+/// The existential deposit. Set to 1/10 of the Connected Relay Chain.
```

Comment:
have you discussed all these proposed values with the team?

Reply:
@ltfschoen I was following this branch https://github.com/DataHighway-DHX/node/blob/ilya/parachain-update/runtime/src/lib.rs#L310. Value was picked initially from `rococo-v1`. I picked these values from https://github.com/substrate-developer-hub/substrate-parachain-template/blob/polkadot-v0.9.11/runtime/src/lib.rs#L207

Reply:
I have made this similar to what we have in master branch.